README.md

```diff
@@ -381,464 +607,224 @@ class MyBaseClass extends WidgetBase<WidgetProperties> {
-import { WidgetBase } from '@dojo/widget-core/WidgetBase';
-import { ThemeableMixin, ThemeableProperties, theme } from '@dojo/widget-core/mixins/Themeable';
-import * as baseClasses from './styles/tabpanel.css';
-
-interface MyThemeableWidgetProperties extends WidgetProperties, ThemeableProperties;
+```tsx
+class MyWidgetWithTsx extends WidgetBase<MyProperties> {
+	protected render(): DNode {
+		const { clear, properties: { completed, count, activeCount, activeFilter } } = this;
 
-@theme(baseClasses)
-class MyThemeableWidget extends ThemeableMixin(WidgetBase)<MyThemeableWidgetProperties> {
-	render: function () {
-		const { root, tab } = baseClasses;
-		return
-			v(`ul`, { classes: this.classes(root) }, [
-				v('li', { classes: this.classes().fixed(tab) }, [ 'tab1' ])
-				// ...
-			]);
+		return (
+			<footer classes={this.classes(css.footer)}>
+				<span classes={this.classes(css.count)}>
+					<strong>{`${activeCount}`}</strong>
+					<span>{`${count}`}</span>
+				</span>
+				<TodoFilter activeFilter={activeFilter} />
+				{ completed ? ( <button onclick={clear} /> ) : ( null ) }
+			</footer>
+		);
 	}
 }
 ```
 
-In the above example, the `root` class is still themeable, but the `tab` class is applied using `.fixed()` so it will not be themeable. The classes passed to `.fixed()` can be any string, and unlike the `.classes()` parameters, `fixed()` css classes do not need to originate from `baseClasses`.
-
-#### Internationalization (i18n)
-
-Widgets can be internationalized by mixing in `@dojo/widget-core/mixins/I18n`.
-[Message bundles](https://github.com/dojo/i18n) are localized by passing them to `localizeBundle`.
-
-If the bundle supports the widget's current locale, but those locale-specific messages have not yet been loaded, then the default messages are returned.
-The widget will be invalidated once the locale-specific messages have been loaded.
-
-Each widget can have its own locale by passing a property - `properties.locale`.
-If no locale is set, then the default locale, as set by [`@dojo/i18n`](https://github.com/dojo/i18n), is assumed.
-
-```ts
-class I18nWidget extends I18nMixin(WidgetBase)<I18nWidgetProperties> {
-	render: function () {
-		// Load the "greetings" messages for the current locale. If the locale-specific
-		// messages have not been loaded yet, then the default messages are returned,
-		// and the widget will be invalidated once the locale-specific messages have
-		// loaded.
-		const messages = this.localizeBundle(greetingsBundle);
-
-		return v('div', { title: messages.hello }, [
-			w(Label, {
-				// Passing a message string to a child widget.
-				label: messages.purchaseItems
-			}),
-			w(Button, {
-				// Passing a formatted message string to a child widget.
-				label: messages.format('itemCount', { count: 2 })
-			})
-		]);
-	}
-}
-```
+**Note:** Unfortunately `tsx` is not directly used within the module so will report as an unused import so would be needed to be ignored by linters.
```

Comment:
"so will report as an unused import so would be needed to be ignored by linters." -> "so the `tsx` module will report as an unused import, and will need to be ignored by linters."